integration-tests/testconfig/ocr2/ocr2.go

```diff
@@ -2,135 +2,41 @@ package ocr2
 
 import (
 	"errors"
-
-	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 )
 
 type Config struct {
-	Soak   *SoakConfig `toml:"Soak"`
-	Load   *Load       `toml:"Load"`
-	Volume *Volume     `toml:"Volume"`
-	Common *Common     `toml:"Common"`
+	Smoke        *SmokeConfig `toml:"Smoke"`
+	NodeCount    *int         `toml:"node_count"`
+	TestDuration *string      `toml:"test_duration"`
```

Comment:
I'd suggest defining it as `blockchain.StrDuration` because you will validation out of the box.

Import this package: `"github.com/smartcontractkit/chainlink-testing-framework/blockchain"`